node_modules/oae-tenants/lib/api.js

```diff
@@ -132,15 +135,26 @@ var init = module.exports.init = function(_serverConfig, callback) {
  * Get a list of all available tenants from cache. The global admin tenant will be excluded from the resulting tenant list
  *
  * @param  {Boolean}        [excludeDisabled]   Whether or not disabled tenants should be included. By default, all tenants will be returned
+ * @param  {Number}         [start]             The index to start paging at, defaults to 0
+ * @param  {Number}         [limit]             The maximum number of tenants to return, if not specified all tenants will be returned
```

Comment:
We should provide a way in which users can search through the list of tenants (matches displayname and host). I'm fine with this being added as a simple `q` parameter on top of the existing endpoint and I'm fine with this being a simple in memory search (if performant enough), but it has to be part of this iteration as an infinitely scrolling tenant list without search is going to be unusable.

If an in-memory search is too slow, we'll have to look at indexing the tenants in Elastic Search ...